packages/api/src/schema/schema.yaml

```diff
@@ -2567,90 +2518,148 @@ components:
 
 paths:
   /stream:
-    tags:
-      - stream
```

Comment:
Hmm what are the tags used for now?

Reply:
Yes, The tags are not really need & doesn't show in the readme.io ui. Do we need them in the schema.yaml? (if yes, will add them back)

Reply:
I think we don'e need them! Feel free to delete them